internal/table/table.go

```diff
@@ -228,13 +166,15 @@ func renderRow(providedColumns []string, colWidths []int) string {
 		maxLen = maxLen - (padding * 2)
 		widths[n] = maxLen
 
-		entries[n] = getCroppedText(columns[n], maxLen)
+		entries[n] = colorize.GetCroppedText(columns[n], maxLen)
 	}
 
 	totalRows := 0
-	for _, columnEntries := range entries {
-		if len(columnEntries) > totalRows {
-			totalRows = len(columnEntries)
+	for _, entry := range entries {
+		// Each entry represents a column. The longest column determines
+		// the number of rows the table will have.
+		if len(entry) > totalRows {
+			totalRows = len(entry)
```

Comment:
Can we rename "rows" to "lines". I think that's what I'm tripping over here. A row can have multiple lines.

Reply:
Ahh okay yeah I can see where the confusion comes from